test/integration/features/provision_earier_version.feature

```diff
@@ -0,0 +1,42 @@
+@minishift-only
```

Comment:
@agajdosi yes deployment take fair amount of time that's why I didn't add it here. I think we are mostly concern here if we can start MiniShift with a older version cleanly. @hferentschik @gbraad  wdyt?